### PR TITLE
GS-250: realtime updates for db user object

### PR DESCRIPTION
**Pull request by gochoa1088**
This PR closes out https://spacem8s.atlassian.net/browse/GS-250

### CODE REVIEW

File: context/UserContext.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      const { uid } = authedUser;
+      const docRef = doc(db, "users", uid);
+      const unsubscribe = onSnapshot(docRef, (doc) => {
+        console.log("listening");
+        setUser(doc.data());
+      });
+      setLoading(false);
+
+      return () => {
+        console.log("unsubscribing");
```
Comment (vincentochoa3) [inline context/UserContext.js:31]: Are we leaving this for future ticket?

Comment (gochoa1088) [reply context/UserContext.js]: console logs removed

File: routers/withUnprotected/index.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { useEffect } from "react";
 import { useRouter } from "next/router";
 
 import { useAuth } from "../../context/AuthContext";
+import { useUser } from "../../context/UserContext";
```
Comment (vincentochoa3) [inline routers/withUnprotected/index.js:5]: Don't think this import is being used

Comment (gochoa1088) [reply routers/withUnprotected/index.js]: removed

**Review by vincentochoa3 — APPROVED** [review]

**Pull request merged**